modules/metastorage/src/testFixtures/java/org/apache/ignite/internal/metastorage/server/SimpleInMemoryKeyValueStorage.java

```diff
@@ -740,22 +741,26 @@ private Collection<Entry> doGetAll(List<byte[]> keys, long rev) {
     }
 
     private Entry doGet(byte[] key, long revUpperBound) {
-        assert revUpperBound >= 0 : "Invalid arguments: [revUpperBound=" + revUpperBound + ']';
```

Comment:
Let's revert this change (information about `revUpperBound` seems to be useful)

Reply:
I tried to fix it differently.